src/CMakeLists.txt

```diff
@@ -865,7 +918,7 @@ if(NOT ALLOCATOR STREQUAL "jemalloc")
 endif()
 add_custom_target(build_rocksdb
     COMMAND
-    PORTABLE=1 ${disable_jemalloc} $(MAKE) static_lib EXTRA_CXXFLAGS='-fPIC -Wno-unused-variable'
+    PORTABLE=1 ${disable_jemalloc} $(MAKE) static_lib V=1 VERBOSE=1 CXX=$(CXX) EXTRA_CXXFLAGS='-Irocksdb/include  -I/usr/local/include -fPIC -Wno-unused-variable'
```

Comment:
@tchaikov 
That might need to go into an if(FREEBSD) box. 
And I'll take care of the V=1 VERBOSE=1 flags in there as well.
I prefer my compilation rather wordy, so I can see what the build system is doing.